geniza/corpus/admin.py

```diff
@@ -82,8 +86,8 @@ class DocumentTextBlockInline(admin.TabularInline):
     model = TextBlock
     autocomplete_fields = ['fragment']
     readonly_fields = ('thumbnail', )
-    fields = ('fragment', 'side', 'extent_label', 'multifragment',
-              'order', 'certain', 'thumbnail')
+    fields = ('thumbnail', 'fragment', 'subfragment', 'side', 'region',
```

Comment:
I noticed that fragments with multiple images look a little funny here because they stack vertically and take up space — not sure if we should do anything at this point, just wanted to flag in case you hadn't run into it. (I'm not sure of an easy solution)